src/amazonica/aws/dynamodbv2.clj

```diff
@@ -38,7 +38,7 @@
 (defn- scalar-attribute-value? [value]
   (and (map? value)
        (= 1 (count value))
-       (= 1 (count (name-of-first-key value)))))
+       (contains? #{:s :b :BOOL :SS :BS :n :NS :l :m :NULL} (name-of-first-key value))))
```

Comment:
Hey! All you need to reproduce is actually in the [README](https://github.com/mcohen01/amazonica#dynamodbv2) - the `:endpoint` key can take a URL as well as a region, so you just use `http://localhost:8000` or wherever DynamoDB-local is running.

The code snippet that demonstrates the bug is nearly identical to the `get-item` call that's demonstrated in that same section of the README - my case just had a single-entry map as a `:key` in the `get-table` call however, as shown below:

```clojure
(get-item cred :table-name "TestTable"
               :key {:id {:s "foo"}})
```

If you use Docker and want a really quick in-memory DynamoDB-local instance up and running on port 8000, you can try this:

```sh
docker run -it --rm -p 8000:8000 massiki/dynamodb-local
```